grunttasks/rev.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('rev', {
+    app: {
+      files: {
+        src: [
+          '<%= staticPath %>/bower_components/**/*.js',
```

Comment:
You can remove this one. All bower_components are handled by the requirejs task.